frontend/sw360-portlet/src/main/webapp/html/licenses/view.jsp

```diff
@@ -6,18 +6,20 @@
   ~ which accompanies this distribution, and is available at
   ~ http://www.eclipse.org/legal/epl-v10.html
   --%>
+<%@ page import="org.eclipse.sw360.portal.common.PortalConstants" %>
```

Comment:
only some reordering to follow the JSP code conventions at http://www.oracle.com/technetwork/articles/java/code-convention-138726.html